versions/app/src/screens/menuDetailScreen.js

```diff
@@ -178,14 +179,12 @@ export default function MenuDetailScreen({ route, navigation }) {
 				<View style={styles.content}>
 					<View style={styles.header}>
 						<View style={styles.title}>
-							<Text style={globalStyles.h1}>{details.title}</Text>
+								<Text style={globalStyles.h1}>{details.title}</Text>
+						</View>
+						<View style={styles.price_cont}>
+								<Text style={styles.price_label}>EPC Suggested Price</Text>
+								<Text style={styles.price}>${details.price}<Text style={styles.price_label}>/Person</Text></Text>
 						</View>
```

Comment:
This should probably all be conditional as a menu does not ***have*** to have a suggested price. I have menus on my account that don't have suggested prices and is simply appearing as `$/person`, which I don't think is the desired effect.

Reply:
Good point. I did update the menu_templates to all have pricing and you're right when we shouldn't assume that we may not have a price set on a menu if the user doesn't know yet.